examples/epac2004_benchmarks/run_fodo_rf_SC.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+#
+# Copyright 2022-2023 ImpactX contributors
+# Authors: Marco Garten, Axel Huebl, Chad Mitchell
+# License: BSD-3-Clause-LBNL
+#
+# -*- coding: utf-8 -*-
+
+import amrex.space3d as amr
+from impactx import ImpactX, RefPart, distribution, elements
+
```

Comment:
Lines 9-10 appear to give a line continuation error.  It's not clear to me why this is the case.

Reply:
Unrelated change appears to resolve this.